Themes/default/Profile.template.php

```diff
@@ -2698,8 +2698,16 @@ function template_error_message()
 
 		// Cycle through each error and display an error message.
 		foreach ($context['post_errors'] as $error)
-			echo '
+			if ($error == 'password_short')
```

Comment:
You can move the check for "password_short" to a ternary to avoid the if/else condition:

```php
		foreach ($context['post_errors'] as $error)
		{
			$text_key_error = $error == 'password_short' ?
				printf($txt['profile_error_' . $error], (empty($modSettings['password_strength']) ? 4 : 8)) :
				$txt['profile_error_' . $error];

				echo '
				<li>', isset($txt['profile_error_' . $error]) ? $text_key_error : $error, '</li>';
		}
```

Reply:
I find that hard to read, but you're the boss. (I think your echo was indented too far, changed that.)